src/mscorlib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -566,7 +566,7 @@ internal static unsafe int CompareOrdinalIgnoreCase(ReadOnlySpan<char> strA, Rea
                 char* b = bp;
 
                 // in InvariantMode we support all range and not only the ascii characters.
-                char maxChar = (char) (GlobalizationMode.Invariant ? 0xFFFF : 0x80);
+                char maxChar = (char) (GlobalizationMode.Invariant ? 0xFFFF : 0x7f);
```

Comment:
Is `GlobalizationMode.Invariant` initrinsicish to the Jit? (like readonly static)

Reply:
should use _invariantMode and not GlobalizationMode.Invariant

Reply:
nit: Change 0x7f to 0x7F (all capital) to be consistent.